packages/core/src/storybook/stand-alone-documentaion/colors/background-colors/background-colors.jsx

```diff
@@ -6,7 +6,8 @@ const colorKeys = [
   "primary-background-hover-color",
   "inverted-color-background",
   "grey-background-color",
-  "allgrey-background-color"
+  "allgrey-background-color",
+  "ui-background-color"
```

Comment:
This was missing here since it was added only in the style storybook, which proves the necessity to have only a single storybook to document it

Reply:
damn